### PR TITLE
Feat/thread sidebar implementation

### PR DESCRIPTION
**Pull request by Ginohmk**
- Configure click on HoverItem component for opening thread
-

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -49,5 +49,8 @@
     "pretty-quick": "^3.1.1",
     "rimraf": "^3.0.2",
     "serve": "^12.0.0"
+  },
```
Comment (ickynavigator) [inline package.json:52]: remove this package from here

File: packages/main/src/components/protected/sidebar/styles/Sidebar.module.css [hunk 2]
```diff
@@ -74,7 +79,9 @@
   margin: 0;
   font-weight: bold;
   font-size: 1.125em;
-  line-height: 28px;
+
+  /* line-height: 28px; */
```
Comment (ickynavigator) [inline packages/main/src/components/protected/sidebar/styles/Sidebar.module.css:83]: remove the commented styles

File: packages/root-config/src/assets/plugin.importmap.json [hunk 3]
```diff
@@ -14,6 +14,6 @@
     "@zuri/zuri-plugin-tools": "https://externaltools.zuri.chat/zuri-zuri-plugin-tools.js",
     "@zuri/zuri-plugin-channels": "https://channels.zuri.chat/static/main/dist/zuri-zuri-plugin-channels.js",
     "@zuri/zuri-plugin-contributions": "https://ct.zuri.chat/js/app.js",
-    "@zuri/zuri-plugin-messaging": "https://chat.zuri.chat/zuri-messaging.js"
+    "@zuri/zuri-plugin-messaging": "http://localhost:5013/zuri-messaging.js"
```
Comment (ickynavigator) [inline packages/root-config/src/assets/plugin.importmap.json:17]: change this back to the original url

File: packages/ui/src/message-board/MessageBoard.styled.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 export const MessageBoardContainer = styled.section`
   display: flex;
   width: 100%;
-  height: 100vh;
+  height: calc(92vh);
```
Comment (ickynavigator) [inline packages/ui/src/message-board/MessageBoard.styled.js:6]: what is this calculating?

File: packages/ui/src/message-pane/components/hover-items/HoverItems.jsx [hunk 5]
```diff
@@ -1,17 +1,16 @@
 import styles from "./hover-items.module.css";
-
 import { Bookmark, More, NewEmoji, Reply, Share } from "@assets/index";
 
 export default function HoverItems(props) {
-  const { id, handleShowMoreOptions, handleShowEmoji } = props;
+  const { id, handleShowMoreOptions, handleShowEmoji, navThrend } = props;
```
Comment (ickynavigator) [inline packages/ui/src/message-pane/components/hover-items/HoverItems.jsx:5]: rename this prop
shouldn't it be `navThread`?

File: packages/ui/package.json [hunk 6]
```diff
@@ -55,13 +55,15 @@
...
     "emoji-mart": "^3.0.1",
+    "eslint-plugin-cypress": "^2.12.1",
     "luxon": "^2.3.0",
     "react": "^17.0.2",
     "react-bootstrap": "^1.6.4",
     "react-click-away-listener": "^2.0.5",
     "react-dom": "^17.0.2",
     "react-giphy-searchbox": "^1.5.4",
     "react-icons": "^4.3.1",
+    "react-router-dom": "6",
```
Comment (ickynavigator) [inline packages/ui/package.json:66]: why is react router dom installed here and why is it a different version

File: packages/ui/package.json [hunk 7]
```diff
@@ -55,13 +55,15 @@
     "draft-js-plugin-editor": "^0.0.0",
     "draft-js-plugins-editor": "^3.0.0",
     "emoji-mart": "^3.0.1",
+    "eslint-plugin-cypress": "^2.12.1",
```
Comment (ickynavigator) [inline packages/ui/package.json:58]: why did you add this plugin?

File: packages/main/package.json [hunk 8]
```diff
@@ -120,7 +120,7 @@
     "react-notifications": "^1.7.2",
     "react-paginate": "^7.1.3",
     "react-redux": "^7.2.5",
-    "react-router-dom": "^5.3.0",
+    "react-router-dom": "5.2.0",
```
Comment (ickynavigator) [inline packages/main/package.json:123]: why is this being downgraded?

**Review by ickynavigator — CHANGES_REQUESTED** [review]

**Comment by JulianaSau** [comment]
@Ginohmk fix your conflicts please and work on the requested changes.
Also, as i stated before, a pr without a proper title and the recommended template will not be merged. Check out other prs to see how they structured their pull request descriptions

**Comment by Ginohmk** [comment]
@JulianaSau , From our chat on slack, another team is implementing this project task. you asked the other team and me, to work together on this. so I am collaborating with them and using their branch now. That being said this branch is no longer used.

**Comment by JulianaSau** [comment]
Okay. Let me close it then

**Pull request closed without merging**